support-frontend/assets/helpers/utilities/utilities.ts

```diff
@@ -55,6 +55,29 @@ function getSanitisedHtml(markdownString: string): string {
 	});
 }
 
+// Automate days remaining copy where campaign has a countdown in the page
+// relies on the countdown updating state in the 3 tier landing page component
+const DEADLINE_PLACEHOLDER_TEMPLATE = '%%CAMPAIGN_DEADLINE%%';
+function replaceDatePlaceholder(copy: string, deadline?: string): string {
+	if (!copy.includes(DEADLINE_PLACEHOLDER_TEMPLATE)) {
+		return copy;
+	}
+
+	if (!deadline || !/^\d+$/.test(deadline)) {
```

Comment:
a minor point, but if we use parseInt first then we have that for validation, e.g.

```
const daysLeft = parseInt(deadline, 10);

if (isNaN(daysLeft)) {
	return copy.replaceAll(DEADLINE_PLACEHOLDER_TEMPLATE, '');
}
```